### ISSUE TITLE
Point 4. What if we train CNN to identify type of tracks?

### ISSUE THREAD

**Issue by FilipsFedotovs** [FilipsFedotovs/EDER-TSU#46]
Create data with CNN training performance. (Validation Loss, Test loss, Accuracy etc. vs Epochs)
